dnsmessage.proto

```diff
@@ -188,6 +188,8 @@ message PBDNSMessage {
   optional uint64 workerId = 25;                // Thread id
   optional bool packetCacheHit = 26;            // Was it a packet cache hit?
   optional uint32 outgoingQueries = 27;         // Number of outgoing queries used to answer the query
+  optional uint16 headerFlags = 28;             // Flags field in wire format
+  optional uint32 ednsVersion = 29;             // EDNS version and flags of first OPT record in wire format
```

Comment:
I don't see anything wrong with specifying we will record the first OPT record seen, but rfc6891 mandates that only one exists in a given message: 
> When an OPT RR is included within any DNS message, it MUST be the only OPT RR in that message.